string attachedCollisionObjectId
---
bool result
string errorCode
